.github/CODEOWNERS

```diff
@@ -383,7 +383,7 @@
 
 # ServiceLabel: %Health Deidentification
 # PRLabel: %Health Deidentification
-/sdk/healthdataaiservices/                                           @GrahamMThomas @alexathomases @Azure/healthdatadeidentification
```

Comment:
intentional?

Reply:
No, good catch. My VS Code is setup to trim trailing spaces yet removed that entire group which is odd since the only content chance was to add the line below.

Reply:
This is fixed and now correct.